doc/tutorial/Tutorial.tex

```diff
@@ -383,14 +378,14 @@ \section{Introduction to the proof engine: Minimal Logic}
 apply H; [ assumption | apply H0; assumption ].
 \end{coq_example}
 
+It is discouraged though to rely on automatically generated names in finished
+proof scripts. And be careful not to abuse \verb:;:.
```

Comment:
I agree with the advice, but any advice is weak if no rationale is given.
Eg. generated names may change and break your scripts; UIs have hard time displaying the intermediate proof state in a composition of many tactics via ;